### DISCUSSION TITLE
Sign In With Ethereum (via Auth0) seems to have died in the transition

### DISCUSSION THREAD

**Discussion by scorpionmit01** [WalletConnect/walletconnect-monorepo#3080] [Developer Support]
Auth0 Supports Sign In With Ethereum

When I connect in to authenticate now, the Wallet Connect QR pops up beautifully (fantastic work). I can scan with Metamask no problem.

However, I just get the following errors about *.bridge.walletconnect.org
I notice in the URL it says protocol=wc&version=1 - which tells me that SIWE (Login.xyz) is still pulling the v1 Bridge.

I cannot tell if the issue is with SIWE or with Auth0 - any help would be highly appreciated.

Actual Errors:

Firefox yields:
Firefox can’t establish a connection to the server at wss://j.bridge.walletconnect.org/?env=browser&host=oidc.login.xyz&protocol=wc&version=1
Chrome yields:
WebSocket connection to 'wss://c.bridge.walletconnect.org/?env=browser&host=oidc.login.xyz&protocol=wc&version=1' failed:

**Comment by crypblizz8** [comment]
Hey there it looks like they are using the V1 bridge as per their wcURI so it is on SIWE to update. 

Will do our best to outreach to them as well.

**Comment by scorpionmit01** [comment]
Okay, fantastic, thank you. Please reach out on your end and I'll push on
my end.

On Tue, Jul 11, 2023 at 12:15 PM crypblizz ***@***.***> wrote:

> Hey there it looks like they are using the V1 bridge as per their wcURI so
> it is on SIWE to update.
>
> Will do our best to outreach to them as well.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/3080#discussioncomment-6418368>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABYN4VOPDMZWRF74IN2EFGDXPV32HANCNFSM6AAAAAA2EX2NJY>
> .
> You are receiving this because you authored the thread.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/3080/comments/6418368
> @github.com>
>


-- 
Book a meeting with me
***@***.***>
